backend/posttest-script.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# stop container
+docker stop test-postgres
+
+# remove container
+docker rm -f test-postgres
+
+# reset NODE_ENV to default
+if [[ "$OSTYPE" == "msys" ]]; then
+  set NODE_ENV=development
```

Shellcheck (0.10.0)

[warning] To assign a variable, use just 'var=value', no 'set ..'.

(SC2121)